tfhe/src/typed_api/integers/tests.rs

```diff
@@ -0,0 +1,27 @@
+use crate::typed_api::{ConfigBuilder, generate_keys, set_server_key, FheUint8};
+use crate::typed_api::prelude::*;
+
+#[test]
+fn test_quickstart_uint8() {
+    let config = ConfigBuilder::all_disabled()
+        .enable_default_uint8()
+        .build();
+
+    let (client_key, server_key) = generate_keys(config);
+
+    set_server_key(server_key);
+
+    let clear_a = 27u8;
+    let clear_b = 128u8;
+
+    let a = FheUint8::encrypt(clear_a, &client_key);
+    let b = FheUint8::encrypt(clear_b, &client_key);
+
+    let result = a + b;
+
+    let decrypted_result: u8 = result.decrypt(&client_key);
+
+    let clear_result = clear_a + clear_b;
+
+    assert_eq!(decrypted_result, clear_result);
+}
```

Comment:
we might be missing a few tests maybe ? Or do we say integer coverage is ok for us ?

Reply:
it's generally ok, though do think we have 500 Gigs of RAM on the test instances, don't spend too much time on this and create an issue for the backlog if there are significant tests we can add in the future